### PR TITLE
Add announce for 2.0.0-rc

### PR DESCRIPTION
**Pull request by AltGr**

### CODE REVIEW

File: opam-2-0-0-rc.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+title: "opam 2.0.0 Release Candidate is out!"
```
Comment (avsm) [inline opam-2-0-0-rc.md]: "Release Candidate 1"?

File: opam-2-0-0-rc.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+authors: "Louis Gesbert" {"mailto:louis.gesbert(à)ocamlpro.com"}
+date: "2018-02-02"
+--BODY--
+
+We are pleased to announce a first release candidate for the long-awaited opam 2.0.0.
+
+A lot of polishing has been done since the last beta, including tweaks to the built-in solver, allowing in-source package definitions to be gathered in an `opam/` directory, and much more.
```
Comment (avsm) [inline opam-2-0-0-rc.md]: link to the previous "last beta" post so users can easily find the previous content

File: opam-2-0-0-rc.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+With all of the 2.0.0 features getting pretty solid, we are now focusing on bringing all the guides up-to-date¹, updating the tools and infrastructure, making sure there are no usability issues with the new workflows, and being future-proof so that further updates break as little as possible.
```
Comment (avsm) [inline opam-2-0-0-rc.md]: did you intend the "1" on up-to-date to be a hyperlink to the footnote?

Comment (AltGr) [reply opam-2-0-0-rc.md]: If you have a clean way to do that with markdown ?

**Pull request merged**